oap-server/server-bootstrap/src/main/resources/agent-dynamic-config.yml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rules:
+  dubbox-provider:
+    trace.sample_rate: 1000
+    trace.ignore_path: /api/seller/seller/*
+  dubbox-consumer:
+    trace.sample_rate: 0
+    trace.ignore_path: /eureka/**
```

Comment:
Why this exists in the `src/main`? This looks like a test file.

Reply:
Give a default value through the configuration file, if others do not use a configuration center like apollo.

Reply:
No, we don't provide that. Any default configuration should be general usage. You are setting for dubbox-provider. Why it makes sense?

Reply:
Ok, I get it now, I will delete the relevant code.